scrunch/datasets.py

```diff
@@ -1572,6 +1573,112 @@ def create_categorical(self, categories, alias, name, multiple, description=''):
             return self.create_single_response(
                 categories, alias=alias, name=name, description=description)
 
+    def get_cube(self, dimensions=[], measures=None, filter=[], weight=None):
```

Comment:
Hey jjdelc. Just saw your comments.

No of cause it doesn't. I just thaught it would make sense since we get a _crunch:cube_ of a _dataset_. But yep - the datasets module is getting quite big so I'll move into a cubes module.